includes/type/object/class-root-query.php

```diff
@@ -497,6 +498,96 @@ public static function register_fields() {
 						return [];
 					},
 				],
+				'collectionStats'      => [
+					'type'        => 'CollectionStats',
+					'args'        => [
+						'calculatePriceRange'  => [
+							'type' => 'Boolean',
+						],
+						'calculateRatingCounts'  => [
+							'type' => 'Boolean',
+						],
+						'taxonomies'  => [
+							'type' => [ 'list_of' => 'CollectionStatsQueryInput' ],
+						],
+					],
+					'description' => __( 'Statistics for a product taxonomy query', 'wp-graphql-woocommerce' ),
+					'resolve'     => function ( $_, $args ) {
+						$data    = [
+							'min_price'           => null,
+							'max_price'           => null,
+							'attribute_counts'    => null,
+							'stock_status_counts' => null,
+							'rating_counts'       => null,
+						];
+						$filters = new ProductQueryFilters();
```

Comment:
I would consider recreating this class inside WooGraphQL plugin so we can modify it accordingly and not be subject to non-existent query params that only make sense in the REST API context.

Reply:
Naw, having gone down that route with the `checkout` mutation. I can say with confidence this is more future-proof and easier to maintain/refactor in the likely event of a breaking change.